src/emqttd_acl_internal.erl

```diff
@@ -53,21 +41,18 @@ all_rules() ->
         [{_, Rules}] -> Rules
     end.
 
-%%%=============================================================================
-%%% ACL callbacks 
-%%%=============================================================================
+%%--------------------------------------------------------------------
+%% ACL callbacks
+%%--------------------------------------------------------------------
 
-%%------------------------------------------------------------------------------
 %% @doc Init internal ACL
-%% @end
-%%------------------------------------------------------------------------------
 -spec init(AclOpts :: list()) -> {ok, State :: any()}.
 init(AclOpts) ->
     ets:new(?ACL_RULE_TAB, [set, public, named_table, {read_concurrency, true}]),
```

Comment:
Line 51 is too long:     ets:new(?ACL_RULE_TAB, [set, public, named_table, {read_concurrency, true}]),.